docs/application/native/guides/text-input/autofill-manager.md

```diff
@@ -0,0 +1,174 @@
+
+   ```xml
+   <privileges>
+      <privilege>http://tizen.org/privilege/autofillmanager</privilege>
+   </privileges>
+   ```
+
+2. To use the functions and data types of the Autofill Manager API, include the `<autofill_manager.h>` header file in your application:
+
+    ```c
+    #include <autofill_manager.h>
+    ```
+
+3. To use the Autofill manager library, create an autofill manager handle.
+
+   The autofill manager handle is used in other autofill manager functions as a parameter.
+
+   ```c
+   void
+   create_autofill_manager_handle()
+   {
+       autofill_manager_h amh;
+       int ret;
+       ret = autofill_manager_create(&amh);
+       if (ret != AUTOFILL_ERROR_NONE)
+           /* Error handling */
+   }
+   ```
+
+4. When you no longer need the autofill manager library, destroy the autofill manager handle using the `autofill_manager_destroy()` function:
+
+   ```c
+   void
+   destroy_autofill_manager_handle(autofill_manager_h amh)
+   {
+       int ret;
+
+       ret = autofill_manager_destroy(amh);
+       if (ret != AUTOFILL_ERROR_NONE)
+           /* Error handling */
+   }
+   ```
+
+   > **Note**
+   >
+   > Do not use the `autofill_manager_destroy()` function in a callback function.
+
+5. After you create the autofill manager handle, connect the background autofill daemon with the `autofill_manager_connect()` function.
+   The function is asynchronous and you can get the result with callback function that registered as the second parameter of `autofill_manager_connect()`.
+
+   ```c
+   static void manager_connection_status_changed_cb(autofill_manager_h amh, autofill_manager_connection_status_e status, void *user_data)
+   {
+       switch (status) {
+           case AUTOFILL_MANAGER_CONNECTION_STATUS_CONNECTED:
+               dlog_print(DLOG_INFO, LOG_TAG, "connected");
+               break;
+           case AUTOFILL_MANAGER_CONNECTION_STATUS_DISCONNECTED:
+               dlog_print(DLOG_INFO, LOG_TAG, "disconnected");
+               break;
+           case AUTOFILL_MANAGER_CONNECTION_STATUS_REJECTED:
+               dlog_print(DLOG_INFO, LOG_TAG, "rejected");
+               break;
+           default:
+               break;
+   }
+
+   void
+   connect_autofill_daemon(autofill_manager_h amh)
+   {
+       int ret;
+       ret = autofill_manager_connect(amh, manager_connection_status_changed_cb, NULL);
+       if (ret != AUTOFILL_ERROR_NONE)
+           /* Error handling */
+   }
+   ```
+
+   You can use the autofill manager functions after receiving `AUTOFILL_MANAGER_CONNECTION_STATUS_CONNECTED` in `manager_connection_status_changed_cb()`.
+
+## Getting the installed autofill service list
+
+To get the installed autofill service list:
+
+1. Create the `autofill_service_info_cb()` callback function, which is invoked for each installed autofill service application and is used to access autofill service information:
```

Comment:
Please change as:
To access Autofill service information, create a callback function using the `autofill_service_info_cb()` function, which is invoked for each installed Autofill service application: